rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml

```
@@ -56,11 +56,12 @@ This rule detects when a remote access tool is seen in the environment for the f
 references = [
     "https://thedfirreport.com/2023/04/03/malicious-iso-file-leads-to-domain-wide-ransomware/",
     "https://attack.mitre.org/techniques/T1219/",
+    "https://github.com/redcanaryco/surveyor/blob/master/definitions/remote-admin.json"
```

```suggestion
    "https://github.com/redcanaryco/surveyor/blob/master/definitions/remote-admin.json",
    "https://www.proofpoint.com/us/blog/threat-insight/security-brief-ta450-uses-embedded-links-pdf-attachments-latest-campaign"
```

Im not sure about having an exhaustive list on malicious RMMs for references. Could be quite expansive.

---

Hmmm wdym?

---

Im suggesting to add the reference link but that is only one example of this. Not sure if we need to grab all examples and list several of them in references. Up to you 😅